src/components/TodoComponents/Todo.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const Todo = props =>  {
+    return (
+        <div style={{ textDecoration: props.todo.completed ? 'line-through' : 'none' , textDecorationColor: props.todo.completed ? 'red': 'none'}}onClick={()=> {props.toggle(props.todo.id)}}>
```

Comment:
Nice styling inline. Do you feel comfortable adding a className this way as well?